types/redux-define/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for redux-define 1.1
+// Project: https://github.com/smeijer/redux-define
+// Definitions by: Joe Barnett <https://github.com/ensconced>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+export as namespace ReduxDefine;
```

Comment:
Why export as namespace?

Reply:
It has UMD distribution, can be used as global script, sample:
https://codepen.io/blazejewicz/pen/GRQMpoo